src/main/resources/edu/cornell/kfs/fp/document/validation/configuration/FinancialProcessingValidators.xml

```diff
@@ -35,6 +35,9 @@
           p:parameterEvaluatorService-ref="parameterEvaluatorService" p:parameterService-ref="parameterService"/>
 	<bean id="DisbursementVoucher-bankCodeValidation" class="edu.cornell.kfs.fp.document.validation.impl.CuDisbursementVoucherBankCodeValidation" abstract="true" />
 	<bean id="DisbursementVoucher-payeeStateAndCountryValidation" class="edu.cornell.kfs.fp.document.validation.impl.CuDisbursementVoucherPayeeStateAndCountryValidation" abstract="true">
+		<property name="recurringDisbursementVoucherForeignVendorValidation" ref="RecurringDisbursementVoucherForeignVendorValidation" />
```

Comment:
"RecurringDisbursementVoucherForeignVendorValidation" is an abstract bean. If this reference is not getting materialized as expected, then you may have to change this to use the \<bean parent="..."/\> setup. (In cases like the latter, you might need to specify it as a prototype bean as well, for consistency with similar spots in KFS's validator beans.)

Reply:
Based on the Spring bean fix that you recently put in for the requisition issue, a possible alternative way to fix this would be to redefine "RecurringDisbursementVoucherForeignVendorValidation" as a non-abstract bean with prototype scope.

Reply:
Yes, I switched to non abstract and scoped it